pages/01.metakube/01.release-notes/default.en.md

```diff
@@ -10,6 +10,10 @@ page-toc:
 <!-- markdownlint-disable MD036 -->
 ## July 2019
 
+**2019-07-18**
+
+* The scheduler is now aware of the maximum number of persistant storage volumes that can be attached to a node, and will try to make sure that pods with mounted volumes are only scheduled on nodes can support them.
```

Comment:
And will not try to do it, it ensures that and if there is no node where there is space the pod stays in pending.
Also this affects only OpenStack and we should mention what the limit is.

Reply:
Updated.